GERMLINE_0001.cpp

```diff
@@ -90,7 +95,7 @@ int main(int argc, char* argv[])
 
 	GERMLINE germline;
     germline.mine( params );
-    return 1;
+    return 0;
```

Comment:
There's a spot in the pipeline where I remember we were handling this bug, so we'll need to fix that.